NEWS

```diff
@@ -1,7 +1,7 @@
 Ticket reference tags refer to tickets in the MongoDB JIRA tracking system:
 https://jira.mongodb.org
 
-WiredTiger release 11.0.1, 2022-XX-XX
+WiredTiger release 11.0.0, 2022-05-12
```

Comment:
Do we still need the NEWS file? The current version of content is duplicated with the `src/docs/upgrading.dox` file. [The previous version of content on master branch](https://github.com/wiredtiger/wiredtiger/blob/master/NEWS) contains the change history for each open-source release version.

Reply:
I'd say "no".
For the current release, it's all out-of-date and we're not going to do the work to create one.

For future releases, assuming we follow the plan of connecting OSS releases with MongoDB drops, then we can point to that information instead, can't we?

Reply:
> For future releases, assuming we follow the plan of connecting OSS releases with MongoDB drops, then we can point to that information instead, can't we?

I'm not sure we'd plan to correlate OSS releases with MDB releases (or any WT drop) - it would be more independent in my understanding. The content we'd like to cover in the NEWS file (or Github version tag page) is merely a Jira link from which one can navigate to see the list of WT tickets included in the release. We used to highlight some significant changes/tickets but that involves some filtering effort that we'd love to save in future releases.

Reply:
My hope was that linking OSS drops to MDB releases would give us a steady drumbeat with minimal effort. If we have to think about when to make the OSS drop, we will not make the OSS drops.